src/material-examples/snack-bar-component/snack-bar-component-example.html

```diff
@@ -1,3 +1,8 @@
+<mat-form-field>
+  <mat-label>Snack bar duration</mat-label>
```

Comment:
`Snack bar duration (seconds)` ?

And then handle the `* 1000` in the code

Reply:
Good point. Changed.